components/alert/AlertMap.vue

```diff
@@ -190,7 +241,7 @@ export default Vue.extend({
       if (this.alert.state === Ps2alertsEventState.ENDED) {
         this.clearTimers()
         this.pull()
-        this.remaining.$el.remove();
+        this.remaining.$el.remove()
```

Comment:
I looked at the other files in the `components/alert/` directory and it looks like they all currently call it `alert` as well, so if we're gonna change it we should probably do it on everything in one PR rather than piecemeal

I went ahead and restructured the DOM to remove the `.remove()` call, I think I was having issues with it previously not working the way I expected when I placed the v-show directly on the element I wanted to remove

Reply:
> probably do it on everything in one PR rather than piecemeal

Yeah agreed

Appriciate the v-show 👍